tokio-postgres/src/lib.rs

```diff
@@ -156,6 +156,7 @@ mod connect_raw;
 mod connect_socket;
 mod connect_tls;
 mod connection;
+mod copy_both;
```

Comment:
You also need a `pub use copy_both::CopyBothDuplex;`

Reply:
fixed